topics/proteomics/tutorials/clinical-mp-data-verification/tutorial.md

```diff
@@ -0,0 +1,406 @@
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `out_file1` (output of **Group** {% icon tool %})
+>            - In *"Filter Dataset Input"*:
+>                - In *"Filter Tabular Input Lines"*:
+>                    - {% icon param-repeat %} *"Insert Filter Tabular Input Lines"*
+>                        - *"Filter By"*: `normalize list columns, replicates row for each item in list`
+>                            - *"enter column numbers to normalize"*: `2`
+>                            - *"List item delimiter in column"*: `;`
+>                    - {% icon param-repeat %} *"Insert Filter Tabular Input Lines"*
+>                        - *"Filter By"*: `regex replace value in column`
+>                            - *"enter column number to replace"*: `2`
+>                            - *"regex pattern"*: `(tr|sp)[|]`
+>                    - {% icon param-repeat %} *"Insert Filter Tabular Input Lines"*
+>                        - *"Filter By"*: `regex replace value in column`
+>                            - *"enter column number to replace"*: `2`
+>                            - *"regex pattern"*: `[ ]+`
+>                    - {% icon param-repeat %} *"Insert Filter Tabular Input Lines"*
+>                        - *"Filter By"*: `regex replace value in column`
+>                            - *"enter column number to replace"*: `2`
+>                            - *"regex pattern"*: `[|].*$`
+>            - In *"Table Options"*:
+>                - *"Specify Name for Table"*: `t1`
+>                - *"Use first line as column names"*: `Yes`
+>                - *"Specify Column Names (comma-separated list)"*: `pep,prot`
+>    ` *"SQL Query to generate tabular output"*: `SELECT distinct(prot) AS Accession 
+>     from t1`
+>     *"include query result column headers"*: `No`
+>
+>
+{: .hands_on}
+
+
+> <question-title></question-title>
+>
+> 1. What is the accession number of a protein?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. An accession number of a protein, also referred to as a protein accession number, is a unique identifier assigned to a specific protein sequence in a protein sequence database. These accession numbers are used to reference and catalog proteins in a standardized and systematic manner
+> >
+> {: .solution}
+>
+{: .question}
+
+> <question-title></question-title>
+>
+> 1. Can there be multiple accession numbers for one peptide or protein?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Yes, a single peptide or protein can have multiple accession numbers, particularly when dealing with different protein sequence databases, databases for specific species, or different versions of the same database. That's the reason in our workflow we merge both accession and sequences.
+> >
+> {: .solution}
+>
+{: .question}
```

Comment:
since these two questions ar both about the same topic, I would group them into a single box with 2 questions in it.